src/skill/getAvailMachinesIntent.py

```diff
@@ -0,0 +1,64 @@
+import boto3
+from utils import *
+from boto3.dynamodb.conditions import Key, Attr
+
+"""Dynamo resource and table"""
+dynamodb = boto3.resource('dynamodb', region_name='us-east-2')
+table = dynamodb.Table('Users')
+
+
+def getAvailMachines(intent, session):
+
+    cardTitle = intent.get('name')
+    sessionAttributes = {}
+    shouldEndSession = False

```

Comment:
I would say that in most cases `shouldEndSession` should be `True`. I could see the session not ending after the welcome intent, but other than that most queries are going to be one off questions/statements, which wouldn't require any additional input afterwards. If the session stays open, the user will need to either wait something like 10 seconds for it to close on its own or explicitly say "Alexa, cancel/stop".